internal/srv/handlers.go

```diff
@@ -80,32 +86,40 @@ func (s *Server) processChange(messages <-chan *message.Message) {
 			} else {
 				lb, err = s.newLoadBalancer(m.SubjectID, m.AdditionalSubjectIDs)
 				if err != nil {
-					s.Logger.Errorw("unable to initialize loadbalancer", "error", err, "messageID", msg.UUID)
-					msg.Nack()
+					s.Logger.Errorw("unable to initialize loadbalancer", "error", err, "messageID", msg.UUID, "subjectID", m.SubjectID.String())
+					if errors.Is(err, lbapi.ErrLBNotfound) {
+						// ack and ignore
+						msg.Ack()
```

Comment:
Is this really the behavior we want? Just because we failed to create a load-balancer, I don't think we want to give up on it.

Reply:
when the msg comes in from lbapi, the lb will be there, but say if the msg is delayed significantly and the user deletes the lb, then we don't want to process something that has already been deleted.

Reply:
Let's give this a shot and then we can correct if we see this behavior is becoming a problem